databricks-sdk-java/src/main/java/com/databricks/sdk/core/http/Request.java

```diff
@@ -74,6 +75,11 @@ public Request withHeader(String key, String value) {
     return this;
   }
 
+  public Request withParam(String key, Object value) {
```

Comment:
Let's not couple this interface to the Apache client if possible. Can we just have an `Optional<Boolean> parameter to configure whether the request should follow redirects or not? We want to have flexibility to choose a different client in the future (e.g. when we drop java 8 support, we can use Java's built-in HTTP client).

Reply:
Thanks for the suggestion, @mgyucht - I added the change now - does it look ok to you?